rust/stackablectl/src/args/namespace.rs

```diff
@@ -0,0 +1,14 @@
+use clap::Args;
+use stackable_cockpit::constants::{DEFAULT_OPERATOR_NAMESPACE, DEFAULT_PRODUCT_NAMESPACE};
+
+#[derive(Debug, Args)]
+#[command(next_help_heading = "Namespace options")]
+pub struct CommonNamespaceArgs {
+    /// Namespace in the cluster used to deploy the operators
+    #[arg(short = 'n', long, default_value = DEFAULT_OPERATOR_NAMESPACE, global = true)]
+    pub operator_namespace: String,
+
+    /// Namespace in the cluster used to deploy the products
```

Comment:
Applies to all occurrences: I think this term is more general and makes more sense on a list command
```suggestion
    /// Namespace where the products (e.g. stacks or demos) are deployed
```